### ISSUE TITLE
Test: Create tests for src/screens/OrgPost/OrgPost.tsx

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa-admin#1648]
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

### PR Acceptance Criteria

- When complete this file must show at least **100%** coverage when merged into the code base. 
- [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa-admin?search=&displayType=list)
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by sahildkun** [comment]
I would like to work on this , Can you assign me this issue ?

**Comment by Cioppolo14** [comment]
@Abhishek0943 If the issue has already been assigned, please don't ask to be assigned. We want everyone to get a chance.

**Comment by omsai11** [comment]
did the tests add?

**Comment by sahildkun** [comment]
u can remove me from the assignment as i am busy with other work and can assign it to some one else . Sorry for inconvinience

**Comment by sahildkun** [comment]
Working on it

**Comment by sahildkun** [comment]
@Cioppolo14  can I know how much time I have to fix this issue as I am currently learning the implementation of tests via jest and react-testing library

**Comment by Cioppolo14** [comment]
@sahildkun How much time do you need? Would 7-10 days be enough?

**Comment by pranshugupta54** [comment]
@Cioppolo14, shouldn't this issue be solved after #1675? Once this page is re-designed, the test would change and should be fixed in re-design PR itself

**Comment by sahildkun** [comment]
> @sahildkun How much time do you need? Would 7-10 days be enough? 

Yes that will be enough and if  there is a redesign happening just let me know so that I can respective tests to monitor new features .

**Comment by pranshugupta54** [comment]
Should be covered after https://github.com/PalisadoesFoundation/talawa-admin/issues/1514 (Re-design)

**Comment by palisadoes** [comment]
Closing. No activity
